tripleblindmarket/private-kit
# [SAF-736] - Separate exposure history calculation between GPS and BT

#### Description:
This PR addresses the SAF-736 ticket requirement that when no data is recorded on a day, it should be presented to the user with a different message. This PR also includes additional work for splitting the exposure history calculation between GPS and BT.

NOTE: This PR is still a bit of a WIP, as I need to find a way to pass the GPS-specific initial set of data to the exposure screen. The initial state is currently set to match the BT logic.


#### Screenshots:
<img width="497" alt="image" src="https://user-images.githubusercontent.com/39654964/86366554-35553f80-bc7b-11ea-8d1c-14f2bbff1e3a.png">



#### How to test:
- Initiate the intersection logic (select any HA from the list of available)
- Wait for the intersection to finish
- Go to Exposure History screen and verify
